packages/core/src/state/NetworkManager.ts

```diff
@@ -136,7 +136,12 @@ export default class NetworkManager implements Manager {
 
   protected getLastReset() {
     const lastReset = this.getState().lastReset;
-    if (typeof lastReset !== 'number') return lastReset.getTime();
+    if (
+      typeof lastReset === 'object' &&
+      typeof lastReset.getTime === 'function'
```

Comment:
this could still throw if `lastReset` is `null`.

Reply:
ah, null is object. dang it. what's a better check for Date?

Reply:
the only robust check for Date is https://npmjs.com/is-date-object. The closest thing to that is the quite-brittle `instanceof Date`.